### PR TITLE
Adop 297 sibling

### PR DESCRIPTION
**Pull request by dilipsamrahmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Added the first couple of screens for sibling section

https://tools.hmcts.net/jira/browse/ADOP-443
https://tools.hmcts.net/jira/browse/ADOP-444

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/steps/sibling/court-order-exists/content.test.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  hint: 'For example, a care order or supervision order. Your adoption agency or social worker can help provide these details. (in Welsh)',
+  errors: {
+    hasPoForSiblings: {
+      required: 'Please answer the question (in welsh)',
+    },
+  },
+};
+
+/* eslint-disable @typescript-eslint/ban-types, @typescript-eslint/no-explicit-any */
+describe('sibling > exists > content', () => {
```
Comment (abidfs) [inline src/main/steps/sibling/court-order-exists/content.test.ts:33]:
```suggestion
describe('sibling > cout-order-exists > content', () => {
```

File: src/main/steps/sibling/court-order-exists/content.test.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  test('should return correct welsh content', () => {
+    generatedContent = generateContent({ ...commonContent, language: 'cy' });
+    expect(generatedContent.section).toEqual(cyContent.section);
+    expect(generatedContent.label).toEqual(cyContent.label);
+    expect(generatedContent.hint).toEqual(cyContent.hint);
+    expect(generatedContent.errors).toEqual(cyContent.errors);
+  });
+
+  test('should contain hasSiblings field', () => {
```
Comment (abidfs) [inline src/main/steps/sibling/court-order-exists/content.test.ts:59]:
```suggestion
  test('should contain hasPoForSiblings field', () => {
```

File: src/main/steps/sibling/court-order-exists/content.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    hasPoForSiblings: {
+      type: 'radios',
+      classes: 'govuk-radios',
+      label: l => l.label,
+      section: l => l.section,
+      hint: l => l.hint,
+      values: [
+        { label: l => l.yes, value: YesNoNotsure.YES },
+        { label: l => l.no, value: YesNoNotsure.NO },
+        { label: l => l.notSure, value: YesNoNotsure.NOT_SURE },
```
Comment (abidfs) [inline src/main/steps/sibling/court-order-exists/content.ts:40]: Did you confirm with BA whether we want not sure reason ?

File: src/main/steps/sibling/exists/content.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    hasSiblings: {
+      type: 'radios',
+      classes: 'govuk-radios',
+      label: l => l.label,
+      section: l => l.section,
+      hint: l => l.hint,
+      values: [
+        { label: l => l.yes, value: YesNoNotsure.YES },
+        { label: l => l.no, value: YesNoNotsure.NO },
+        { label: l => l.notSure, value: YesNoNotsure.NOT_SURE },
```
Comment (abidfs) [inline src/main/steps/sibling/exists/content.ts:40]: Confirm with BA if we want to capture not sure reason

File: src/main/steps/sibling/siblingSequence.test.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { YesNoNotsure } from '../../app/case/definition';
+
+import { siblingSequence } from './siblingSequence';
+
+describe('birthMotherSequence', () => {
```
Comment (abidfs) [inline src/main/steps/sibling/siblingSequence.test.ts:5]:
```suggestion
describe('siblingSequence', () => {
```

File: src/main/steps/urls.ts [hunk 6]
```diff
@@ -55,6 +55,17 @@ export const CHILDREN_SEX_AT_BIRTH: PageLink = `${CHILDREN}/sex-at-birth`;
...
 export const CHILDREN_FULL_NAME_AFTER_ADOPTION: PageLink = `${CHILDREN}/full-name-after-adoption`;
 
+export const SIBLING: PageLink = '/sibling';
+export const SIBLING_EXISTS: PageLink = `${SIBLING}/exists`;
+export const SIBLING_COURT_ORDER_EXISTS: PageLink = `${SIBLING}/court-order-exists`;
+export const SIBLING_NAME: PageLink = `${SIBLING}/name`;
+export const SIBLING_ORDER_TYPE: PageLink = `${SIBLING}/order-type`;
+export const SIBLING_ORDER_CASE_NUMBER: PageLink = `${SIBLING}/case-number`;
+export const SIBLING_ORDER_COURT: PageLink = `${SIBLING}/order-court`;
+export const SIBLING_ORDER_DATE: PageLink = `${SIBLING}/order-date`;
```
Comment (abidfs) [inline src/main/steps/urls.ts:65]: order-date does not sound right
can you think of a better url

placement-order-date may be 🤷

**Review by abidfs — CHANGES_REQUESTED** [review]
Few minor comments

File: src/main/steps/sibling/court-order-exists/content.ts (hunk 3 above)
Comment (dilipsamrahmcts) [reply src/main/steps/sibling/court-order-exists/content.ts:40]: confirmed yesterday that this is expected behaviour

File: src/main/steps/urls.ts [hunk 7]
```diff
@@ -55,6 +55,17 @@ export const CHILDREN_SEX_AT_BIRTH: PageLink = `${CHILDREN}/sex-at-birth`;
 export const CHILDREN_NATIONALITY: PageLink = `${CHILDREN}/nationality`;
 export const CHILDREN_FULL_NAME_AFTER_ADOPTION: PageLink = `${CHILDREN}/full-name-after-adoption`;
 
+export const SIBLING: PageLink = '/sibling';
+export const SIBLING_EXISTS: PageLink = `${SIBLING}/exists`;
+export const SIBLING_COURT_ORDER_EXISTS: PageLink = `${SIBLING}/court-order-exists`;
+export const SIBLING_NAME: PageLink = `${SIBLING}/name`;
+export const SIBLING_ORDER_TYPE: PageLink = `${SIBLING}/order-type`;
+export const SIBLING_ORDER_CASE_NUMBER: PageLink = `${SIBLING}/case-number`;
+export const SIBLING_ORDER_COURT: PageLink = `${SIBLING}/order-court`;
```
Comment (abidfs) [inline src/main/steps/urls.ts:64]: URLs are visible to users hence it must be clear 
below URLs can be updated to
```suggestion
export const SIBLING_ORDER_TYPE: PageLink = `${SIBLING}/placement-order-type`;
export const SIBLING_ORDER_CASE_NUMBER: PageLink = `${SIBLING}/placement-order-number`;
export const SIBLING_ORDER_COURT: PageLink = `${SIBLING}/placement-order-court`;
```

**Review by abidfs — CHANGES_REQUESTED** [review]
one last comment

**Review by abidfs — DISMISSED** [review]

**Review by abidfs — APPROVED** [review]

**Pull request merged**